src/components/content/policies/Policies.tsx

```diff
@@ -0,0 +1,297 @@
+        },
+        {
+            title: 'CSP',
+            dataIndex: 'csp',
+            filters: cspFilters,
+            filterMode: 'tree',
+            filterSearch: true,
+            onFilter: (value: string | number | boolean, record) => record.csp.startsWith(value.toString()),
+            render: (csp: AbstractCredentialInfo.csp, _) => {
+                return (
+                    <Space size='middle'>
+                        <Image
+                            width={100}
+                            preview={false}
+                            src={cspMap.get(csp.valueOf() as CloudServiceProvider.name)?.logo}
+                        />
+                    </Space>
+                );
+            },
+        },
+        {
+            title: 'CreateOn',
+            dataIndex: 'createTime',
+            defaultSortOrder: 'descend',
+            sorter: (a, b) => a.createTime.length - b.createTime.length,
+            sortDirections: ['descend'],
+        },
+        {
+            title: 'Enabled',
+            dataIndex: 'enabled',
+            render: (text, _) => {
+                return text ? 'true' : 'false';
+            },
+        },
+        {
+            title: 'Content',
+            dataIndex: 'policy',
+            render: (_text: string, record: PolicyVo) => {
+                return (
+                    <Popover
+                        content={
+                            <pre>
+                                {' '}
+                                <div className={'policy-content-hover'}>{record.policy.toString()}</div>
+                            </pre>
+                        }
+                        title={'Policy Content'}
+                        trigger='hover'
+                    >
+                        <Button className={'policy-data-hover'} type={'link'}>{`Policy Content`}</Button>
+                    </Popover>
+                );
+            },
+        },
+        {
+            title: 'Operation',
+            dataIndex: 'operation',
+            render: (_text: string, record: PolicyVo) => {
+                return (
+                    <>
+                        <Space size='middle'>
+                            <Button
+                                type='primary'
+                                icon={<InfoCircleOutlined />}
+                                onClick={() => {
+                                    showPoliciesManagementServiceDetails(record);
+                                }}
+                                disabled={
+                                    (deletePoliciesManagementServiceRequest.isPending ||
+                                        deletePoliciesManagementServiceRequest.isSuccess ||
+                                        deletePoliciesManagementServiceRequest.isError) &&
+                                    !isDeletingCompleted
+                                }
+                            >
+                                details
```

Comment:
we do not need the 'details' option. Please remove it. It has no extra information.